tests/SparkFabrik/Command/Redmine/RedmineSearchCommandTest.php

```diff
@@ -421,4 +468,172 @@ public function testSearchByMoreThanOneStatus()
+            $this->createMocks(
+                array(
+                    'redmineApiIssueAll' => unserialize($response_mock),
+                )
+            );
+
+            $input = array(
+                'command' => $this->command->getName(),
+                '--project_id' => 'test_project_id',
+                '--assigned' => 1,
+                '--fields' => 'id'
+            );
+
+            $this->tester->execute($input);
+            $res = trim($this->tester->getDisplay());
+            $expected = <<<EOF
++------+
+| ID   |
++------+
+| 8921 |
++------+
+EOF
+            ;
+            $this->assertContains($expected, $res);
+        }
+
+    /**
+     * Test search by wrong assigned user.
+     */
+        public function testSearchByAssignedUserNotFound()
+        {
+            $response_mock = file_get_contents(self::$fixturesPath . 'response_one_issue_assigned_user.serialized');
+            $current_user_admin_mock = file_get_contents(self::$fixturesPath . 'response_current_user_admin.serialized');
+            $users_mock = file_get_contents(self::$fixturesPath . 'response_users_user.serialized');
+
+            $command = $this->createCommand('redmine:search');
+            $this->createMocks(
+                array(
+                    'redmineApiUserGetCurrentUser' => unserialize($current_user_admin_mock),
+                    'redmineApiIssueAll' => unserialize($response_mock),
+                    'redmineApiUserAll' => unserialize($users_mock),
+                )
+            );
+
+            $input = array(
+                'command' => $this->command->getName(),
+                '--project_id' => 'test_project_id',
+                '--assigned' => 'WRONG NAME'
+            );
+
+            $this->tester->execute($input);
+            $res = trim($this->tester->getDisplay());
+            $expected = "No user found.";
+            $this->assertContains($expected, $res);
+        }
+
+    /**
+     * Test search by assigned user from not admin user without specify project.
+     */
+        public function testSearchByAssignedUserFromNotAdminUserWithoutProjectId()
+        {
+            $current_user_not_admin_mock = file_get_contents(self::$fixturesPath . 'response_current_user_not_admin.serialized');
+
+            $command = $this->createCommand('redmine:search');
+            $this->createMocks(
+                array(
+                    'redmineApiUserGetCurrentUser' => unserialize($current_user_not_admin_mock),
+                )
+            );
+
+            $input = array(
+                'command' => $this->command->getName(),
+                '--project_id' => '',
+                '--assigned' => 'Paolo Pustorino'
+            );
+
+            $this->tester->execute($input);
+            $res = trim($this->tester->getDisplay());
+            $expected = "To perform search by assigned user specify the project id.";
+            $this->assertContains($expected, $res);
```

Comment:
It's not possible convert using @expectedexception and @excectedexceptionmessage because this case throw exception in internal method managed as print in the final catch. In this case the assertion is relative to the right error message